internal/cmd/tracking-issue/main.go

```diff
@@ -165,9 +179,19 @@ type Workload struct {
 	Days         float64
 	Issues       []*Issue
 	PullRequests []*PullRequest
+	Labels       []string
 }
 
-func (wl *Workload) Markdown() string {
+func (wl *Workload) AddIssue(newIssue *Issue) {
+	for _, issue := range wl.Issues {
+		if issue.URL == newIssue.URL {
```

Comment:
This should be equivalent and marginally more efficient (i.e. integer comparison).
```suggestion
		if issue.ID == newIssue.ID {
```

Reply:
for clarification: the ID is a string field, i think an internal canonical ID github uses, rather than the Number (e.g. #10082). we can't use the Number because it doesn't work across repositories. between using the URL and ID as canonical issue representations, i still prefer using the URL since we are already using it when parsing the body and we can construct it ourselves from issue meta data (org, repo, number). we could also write a comparison function that directly compares that metadata, but the URL can be used as a map key and and I've been favouring code simplicity for this diff.

i lean lightly towards the URL given all that but am happy with something else if you think it makes more sense.

Reply:
Sounds good to me.